playground/AzureServiceBus/ServiceBus.AppHost/Program.cs

```diff
@@ -0,0 +1,47 @@
+using System.Text.Json.Nodes;
+using Aspire.Hosting.Azure.ServiceBus;
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+var serviceBus = builder.AddAzureServiceBus("sbemulator");
+
+serviceBus
+    .WithQueue("queue1", queue =>
+    {
+        queue.DeadLetteringOnMessageExpiration = false;
+    })
+    .WithTopic("topic1", topic =>
+    {
+        var subscription = new ServiceBusSubscription("sub1")
+        {
+            MaxDeliveryCount = 10,
+        };
+        topic.Subscriptions.Add(subscription);
+
+        var rule = new ServiceBusRule("app-prop-filter-1")
+        {
+            CorrelationFilter = new()
+            {
+                ContentType = "application/text",
+                CorrelationId = "id1",
+                Subject = "subject1",
+                MessageId = "msgid1",
+                ReplyTo = "someQueue",
+                ReplyToSessionId = "sessionId",
+                SessionId = "session1",
+                SendTo = "xyz"
+            }
+        };
+        subscription.Rules.Add(rule);
+    })
+    ;
+
+serviceBus.RunAsEmulator(configure => configure.ConfigureJson(document =>
```

Comment:
Can we think of a better name than `ConfigureJson` here? That doesn't seem very descriptive of what it is doing.

What does the emulator call the json file? Maybe `ConfigureEmulatorConfiguration`? (maybe that's too many configure words...)

Reply:
> What does the emulator call the json file

They call it:

> the [config.json](https://github.com/Azure/azure-service-bus-emulator-installer/blob/main/ServiceBus-Emulator/Config/Config.json) configuration file.

And 

> supply any changes in JSON configuration before you run the emulator

Reply:
Changing to `WithConfigurationFile` when it takes a file and `ConfigureEmulator` when it takes an `Action<JsonNode>`